test/multipart.test.js

```diff
@@ -380,7 +380,7 @@ if (!process.env.TRAVIS) {
 }
 
 test('should not allow __proto__', function (t) {
-  t.plan(5)
+  t.plan(3)
```

Comment:
why this change was needed?

Reply:
this test doesn't pass on my machine because of the incorrect plan. I have counted assertations by hand and counted 3 assertations. Now the test is passing)

Reply:
I counted and I see 5 assertions:

https://github.com/fastify/fastify-multipart/blob/d298f193dc4e0d0a446fae70789ceeafc05e3211/test/multipart.test.js#L391

https://github.com/fastify/fastify-multipart/blob/d298f193dc4e0d0a446fae70789ceeafc05e3211/test/multipart.test.js#L394

https://github.com/fastify/fastify-multipart/blob/d298f193dc4e0d0a446fae70789ceeafc05e3211/test/multipart.test.js#L420

https://github.com/fastify/fastify-multipart/blob/d298f193dc4e0d0a446fae70789ceeafc05e3211/test/multipart.test.js#L423

https://github.com/fastify/fastify-multipart/blob/d298f193dc4e0d0a446fae70789ceeafc05e3211/test/multipart.test.js#L431

Could you retry to run the test on master in your PC?
I'm afraid the changes do not cover some edge cases

Reply:
This is my `npm t` output when `t.plan(5)`:

```
test/append-body.test.js ............................ 65/65
test/multipart.test.js .............................. 53/55 30s
  should not allow __proto__
  not ok test count !== plan
    --- wanted
    +++ found
    -5
    +3
    results:
      ok: false
      count: 3
      pass: 3
      fail: 1
      bailout: false
      todo: 0
      skip: 0
      plan:
        start: 1
        end: 5
        skipAll: false
        skipReason: ''
        comment: ''
      failures:
        - tapError: incorrect number of tests

  not ok test count !== plan
    --- wanted
    +++ found
    -1
    +6
    results:
      ok: false
      count: 6
      pass: 6
      fail: 1
      bailout: false
      todo: 0
      skip: 0
      plan:
        start: null
        end: null
        skipAll: false
        skipReason: ''
        comment: ''
      failures:
        - tapError: no plan

total ............................................. 118/120


  118 passing (33s)
  2 failing
```

According to Tap docs `t.pass` is not an assertion, but `t.error` is. Not sure why this is happening on my machine

Reply:
Try to commit the plan 5 (`git commit -nm 'msg'`), I would like to check CI output

Reply:
`t.pass` is an assertion as the docs state it is literally the basis for all success assertions:

https://node-tap.org/docs/api/#tpassmessage
> Emit a passing test point. This method, and `fail()`, are the basic building blocks of all fancier assertions.

Reply:
Ok, I see tests are passing on CI now. It's possible to merge this PR now
I think it's a very strange behaviour of TAP. Perhaps it depends on the OS or node version